### ISSUE TITLE
LANGUAGE

### ISSUE THREAD

**Issue by bogdomania** [oragono/oragono#803]
To use one of the available translations, one must use /language <code>.

Good ideea, poor implementation, imo, for current version. <code> should keep country extension,
since en_WHATEVA is confusing.
ro-WHATEVA, es-ETC.

Just /language us, /language gb(or en), /language au, /language es, /language mx, etc.

**Comment by DanielOaks** [comment]
Yeah it's a little annoying. I wonder what ways we could simplify this, right now we rely exclusively on recognised language identifier codes so that e.g. programs with actual interfaces built for the feature can very easily provide a nice gui to work with. Are there any official/published 'simple mappings' or something to the actual language codes, maybe we need to expand how 'smart' the language-code-matching in the ircd tries to be... etc?

**Comment by bogdomania** [comment]
Well, I'd say take as a starting point this [https://en.wikipedia.org/wiki/List_of_official_languages_by_country_and_territory](url),
and see from there on.. If a state has lots of minorities, that's a niche. IMO let's focus on major/recognised languages. How to implement them on ORA? I don't have a clue, yet.

**Comment by DanielOaks** [comment]
Yeah, unfortunately we need something that client devs will be referencing and that'll stay consistent over time which is why we currently refer to the official language codes. If we want to provide a much easier interface for this that's more human-compatible, I'd look at making some NickServ interface that can smooth out the usability.
